css/style.css

```diff
@@ -0,0 +1,10 @@
+body{
+    width: 100%;
+    display: flex;
+    justify-content: center;
+}
+
+#image{
+    width: 100%;
```

Comment:
In order to make the document visually better. It would be a good idea to justify the content to the center and add some margins to the top and bottom.